extension/chrome/src/js/injector_loader.js

```diff
@@ -1,71 +1,55 @@
-var isBitbucket = false;
-var extensionId = chrome.runtime.id || 'stashFF';
+let isBitbucket = false;
+const extensionId = chrome.runtime.id || 'stashFF';
```

Comment:
is this file executed after `background.js`?

there is a line `chrome = chrome || browser` and `browser.runtime.id` exists - https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/runtime/id